### ISSUE TITLE
[BUG] Constand "Write after end" errors in Developer Console.

### ISSUE THREAD

**Issue by jtsom** [microsoft/vscode-dotnettools#1328]
### Describe the Issue

When selecting any Razor file in a .Net 8 Blazor app, there is an error reported in the console:

![image](https://github.com/user-attachments/assets/23c57c7c-fccb-4eb3-b463-f1bce0ba9610)

### Steps To Reproduce

1. In a .Net 8 Blazor application, click on any .razor file to show in the editing window.
2. Look in the developer console.
3. See the error above for every .razor file as they are clicked.

### Expected Behavior

No Errors.

### Environment Information

- OS: Windows 10
- VSCode: 
```
Version: 1.92.0-insider (user setup)
Commit: 3bb157866cdb10b90e5f5999e1d9855c294aa7c7
Date: 2024-07-31T12:31:58.409Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631
```
C# Extension: v2.41.26 (pre-release)
C# Dev Kit: v1.9.53 (pre-release)

**Comment by AbhitejJohn** [comment]
Tagging @phil-allen-msft  since this seems blocking.

**Comment by ryzngard** [comment]
This doesn't look related to razor at all, it's the C# Dev Kit extension reporting an error. 

Are you seeing any impact on features from this? This doesn't appear for me locally but the current pre-release versions are higher than the ones used.

**Comment by rjgotten** [comment]
> @ryzngard 
> Are you seeing any impact on features from this?

See #1371 - where I'm also getting this.
In addition to a lot of bogus errors regarding unkown types as reported in that issue, I'm also stuck having to filter out approx 500 non-existent errors from `*.cshtml__virtual.cs` files generated by Razor. So the two may in fact be connected.

**Comment by jtsom** [comment]
> > @ryzngard
> > Are you seeing any impact on features from this?
> 
> See #1371 - where I'm also getting this. In addition to a lot of bogus errors regarding unkown types as reported in that issue, I'm also stuck having to filter out approx 500 non-existent errors from `*.cshtml__virtual.cs` files generated by Razor (i.e. plain old MVC Razor, not Blazor). So the two may in fact be connected.

Agreed. This issue with the non-existent errors I reported months ago and it was punted from the VSCode C# extension to the Razor group, and there's been nothing done. (As far as I could tell, the problem began when the C# extension was changed to being built with the .Net 9 preview SDK. Installing an older version of the C# extension (I forget which version, but may be back to 2.32.16?) cleared up the error. You can see, looking at the release notes, the first version built with .Net 9, is when it broke.

**Comment by ryzngard** [comment]
The errors in this issue are if you open the developer console for vs code, not errors reported in the "Problems" output. 

@rjgotten I didn't see cshtml mentioned in https://github.com/microsoft/vscode-dotnettools/issues/1371 . Are you seeing these problems only in MVC Razor?

@jtsom I'm assuming you're talking about https://github.com/dotnet/razor/issues/10505 ?

**Comment by rjgotten** [comment]
> The errors in this issue are if you open the developer console for vs code, not errors reported in the "Problems" output.
> 
> @rjgotten I didn't see cshtml mentioned in #1371 . Are you seeing these problems only in MVC Razor?
> 
> @jtsom I'm assuming you're talking about [dotnet/razor#10505](https://github.com/dotnet/razor/issues/10505) ?

I hadn't gotten round to filing the `*.cshtml__virtual.cs` specific errors.
But those are only in the range of ~500 or so specific to classes being used in `.cshtml` razor files.
Whereas the full number of 20K+ bogus errors wrt missing types largely comes from throughout the entire codebase mentioned in #1371 - i.e. also regular old C# 'library' projects.

**Comment by ryzngard** [comment]
Closing this issue. If there is a product issue outside of developer console errors please file that separately. For diagnostics we have a fix that will be coming in the next release (and is currently in prerelease)
